inference-engine/tests/functional/shared_test_classes/src/base/ov_layer_test_utils.cpp

```diff
@@ -0,0 +1,317 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <fstream>
+#include <signal.h>
+#ifdef _WIN32
+#include <process.h>
+#endif
+
+#include <transformations/serialize.hpp>
+#include <transformations/op_conversions/convert_batch_to_space.hpp>
+#include <transformations/op_conversions/convert_space_to_batch.hpp>
+#include <ngraph/opsets/opset.hpp>
+#include <pugixml.hpp>
+#include <common_test_utils/file_utils.hpp>
+#include <thread>
+
+#include "openvino/core/variant.hpp"
+#include "shared_test_classes/base/ov_layer_test_utils.hpp"
+#include "functional_test_utils/ov_tensor_utils.hpp"
+
+namespace ov {
+namespace test {
+
+LayerTestsCommon::LayerTestsCommon() {
+    core = *PluginCache::get().core(targetDevice);
+}
+
+void LayerTestsCommon::run() {
+    auto crashHandler = [](int errCode) {
```

Comment:
Please, do not lose `SKIP_IF_CURRENT_TEST_IS_DISABLED();` in the `Run()` function